documentation/modules/ROOT/pages/configuration/signalling.adoc

```diff
@@ -15,159 +17,275 @@ This feature is currently in incubating state, i.e. exact semantics, configurati
 ====
 
 == Overview
-Sometimes it is necessary for the user to modify a connector behaviour or trigger a one-time action (e.g. snapshot of a single table).
-To fulfill such needs {prodname} provides a mechanism how to send a signal to a {prodname} connector to trigger an action.
-As it might be necessary to synchronize such an action with the dataflow the signal is implemented as a write to a data collection.
-
-Signalling is disabled by default.
-The name of the signalling data collection must be set via connector configuration parameter to enable the function.
-The signalling data collection must be *explictly* added among captured data collections, i.e., included in the `table.include.list` parameter.
+endif::community[]
 
-.Connector configuration
-[cols="3,9",options="header"]
+ifdef::product[]
+[IMPORTANT]
+====
+Signaling is a Technology Preview feature.
+Technology Preview features are not supported with Red Hat production service-level agreements (SLAs) and might not be functionally complete;
+therefore, Red Hat does not recommend implementing any Technology Preview features in production environments.
+This Technology Preview feature provides early access to upcoming product innovations, enabling you to test functionality and provide feedback during the development process.
+For more information about support scope, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
+====
+endif::product[]
+
+The {prodname} signaling mechanism provides a way to modify the behavior of a connector, or to trigger a one-time action, such as initiating an xref:debezium-signaling-ad-hoc-snapshots[ad hoc snapshot] of a table.
+To trigger a connector to perform a specified action, you issue a SQL command to add a signal message to a specialized signaling table, also referred to as a signaling data collection.
+The signaling table, which you create on the source database, is designated exclusively for communicating with {prodname}.
+When {prodname} detects that a new xref:debezium-signaling-example-of-a-logging-record[logging record] or xref:debezium-signaling-example-of-an-ad-hoc-signal-record[ad hoc snapshot record] is added to the signaling table, it reads the signal, and initiates the requested operation.
+
+Signaling is available for use with the following {prodname} connectors:
+
+* Db2
+* MySQL
+ifdef::community[]
+* Oracle
```

Comment:
OK, I'll add. It wasn't on my radar, since it isn't included downstream for q1, but if it's not a problem to include it now and exclude it from the backport to 1.7, I'll make the change now.

Reply:
+1